terraform/vpc.tf

```diff
@@ -0,0 +1,47 @@
+module "vpc" {
+  // Keeping the 'registry.terraform.io/' prefix so Rider can find the module for autocompletion purposes
+  source  = "registry.terraform.io/terraform-aws-modules/vpc/aws"
+  version = "3.14.2"
+
+  name = join("-", ["gpg-application-vpc", var.env])
+  cidr = "10.0.0.0/16"
+
+  azs              = slice(data.aws_availability_zones.available.zone_ids, 0, 2)
+  public_subnets   = [cidrsubnet("10.0.0.0/16", 4, 0), cidrsubnet("10.0.0.0/16", 4, 1)]
+  database_subnets = [cidrsubnet("10.0.0.0/16", 4, 2), cidrsubnet("10.0.0.0/16", 4, 3)]
+
+  enable_ipv6                     = true
+  assign_ipv6_address_on_creation = true
+
+  database_subnet_assign_ipv6_address_on_creation = false
+
+  public_subnet_ipv6_prefixes   = [0, 1]
+  database_subnet_ipv6_prefixes = [2, 3]
+
+  enable_dns_hostnames = true
+  enable_dns_support   = true
+
+  enable_nat_gateway                     = false
+  single_nat_gateway                     = true
```

Comment:
Don't these contradict each other? I don't think we'll be creating any NAT gateways with `enable_nat_gateway` set to `false`